GravatarApp/CommonViews/ModalPresentation/ModalView.swift

```diff
@@ -0,0 +1,42 @@
+import SwiftUI
+
+struct ModalView<Content: View>: View {
+    @ViewBuilder let content: () -> Content
+
+    @State var contentHeight: CGFloat = 0
+
+    var body: some View {
+        ScrollView {
+            content()
+                .contentHeightReader($contentHeight)
+        }
+        .scrollBounceBehavior(.basedOnSize)
+        .background(.regularMaterial)
+        // Render solid color background only on snapshot tests.
+        // Background materials are not rended.
+        .if(ProcessInfo.processInfo.isSnapshotTesting) { view in
+            view.background(
+                Color(uiColor: .secondarySystemBackground)
+            )
+        }
```

Comment:
It's not too bad but for this specific case I would suggest an environment variable like this:

https://github.com/Automattic/Gravatar-iOS/compare/etoledom/main-menu...wppinar/modal-background-environment-var?expand=1

WDYT?

(One snapshot changes because it was applying the .background(...) after another .background(...) and now there's only 1 .background(...) I think)